Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/BaseClasses/CurrentCalculationD.mo

```diff
@@ -0,0 +1,24 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.InductionMotors.BaseClasses;
+function CurrentCalculationD   "Function to compute current to interface with electrical grid"

```

Comment:
The comment `"Function to compute current to interface with electrical grid"` is used by functions `CurrentCalculationD`, `CurrentCalculationD_VFD`, `CurrentCalculationQ`, and `CurrentCalculationQ_VFD`.

Please change the comments to the user how the function is doing.

Also, in the function name, there is no need to have `Calculation`.